app/src/pages/OnDeviceDisplay/ProtocolDashboard/LongPressModal.tsx

```diff
@@ -102,7 +102,10 @@ export function LongPressModal(props: {
   return (
     <>
       {showMaxPinsAlert ? (
-        <TooManyPinsModal
+        <SmallModalChildren
+          header={t('too_many_pins_header')}
+          subText={t('too_many_pins_body')}
+          buttonText={t('shared:close')}
```

Comment:
shared close is `close` so we need to make the first letter capitalize.
```suggestion
          buttonText={i18n.format(t('close'), 'sentenceCase')}
```